src/main/java/de/tum/cit/aet/artemis/assessment/dto/FeedbackDetailDTO.java

```diff
@@ -3,5 +3,5 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public record FeedbackDetailDTO(long count, double relativeCount, String detailText, String testCaseName, int taskNumber) {
+public record FeedbackDetailDTO(long count, double relativeCount, String detailText, String testCaseName, String taskNumber, String errorCategory) {
```

Comment:
Why is the taskNumber now a String?

Reply:
as we do not have a dedicated attribute for the tasknumber i fetch taskname in the query and then map those in further steps to a number. As i am going to display them only on client side therefore i decided to keep it a string.